Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -351,35 +302,61 @@ public extension UIView {
         }
     }
 
-    /// SwifterSwift: Rotate view by angle on relative axis.
+    /// SwifterSwift: Rotate view by radians on relative axis.
     ///
     /// - Parameters:
-    ///   - angle: angle to rotate view by.
-    ///   - type: type of the rotation angle.
-    ///   - animated: set true to animate rotation (default is true).
+    ///   - radian: radians to rotate view by.
+    ///   - animated: set true to animate rotation (default is false).
     ///   - duration: animation duration in seconds (default is 1 second).
     ///   - completion: optional completion handler to run with animation finishes (default is nil).
-    func rotate(byAngle angle: CGFloat, ofType type: AngleUnit, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
-        let angleWithType = (type == .degrees) ? .pi * angle / 180.0 : angle
+    func rotate(byRadian radian: CGFloat, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
```

Comment:
How do we feel about `Measurement<UnitAngle>`? Unfortunately the conversion between radians and degrees isn't exactly correct (don't ask me why, it's the simplest freaking calculation in the world), but there you have it...
```suggestion
    func rotate(byAngle angle: Measurement<UnitAngle>, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
```
I guess the only issue is that `Measurement` is only available from iOS 10+ and we're supporting back to iOS 8, so we'd need 2 implementations.

If you don't want to go for this option, can you please change the parameter name to `radians`? It should be pluralised.

Reply:
@guykogus haven't written out the unit tests yet, but wrote the function that uses Measurement<UnitAngle>.  Totally didn't know about `Measurement` at all.

Before I proceeded anymore, i was wondering the following:

What is the procedure on deprecation / removal? 

I ask because I was wondering if I should just re-add the original functions (that used enum) and deprecate them while also keeping the work i already did and have these new `Measurement` based functions as well.  

I don't want to create unnecessary amount of clutter in the functions, and wasn't sure how SwifterSwift normally would go about doing this.  

I added an additional commit that deprecated my functions i was trying to add to move away from enums, but it seems weird to add deprecated functions in with the PR.  But at the same time, in terms of transitional process I was thinking:  We'd want to mark the enum functions deprecated and get them using the newer deprecated functions if < iOS 10 or the `Measurement` based functions >= iOS 10.  So, enum functions removed first in some build, then when, eventually, SwifterSwift min is iOS 10, we drop the others.

Just my thoughts. It may be too much work this way.